### PR TITLE
Add a connection welcome

### PR DESCRIPTION
**Pull request by juliens**
With --banner option, you can add a welcome banner when the client connect to the whoami (even if the client didn't sent any bytes)

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -52,6 +53,9 @@ func main() {
 		if err != nil {
 			log.Fatal(err)
 		}
+		if banner {
+			conn.Write([]byte("Welcome"))
```
Comment (dduportal) [inline app.go]:
```suggestion
			_, err := conn.Write([]byte("Welcome"))
			if err != nil {
			    log.Fatal(err)
			}			    

```

Comment (dduportal) [reply app.go]: Or `golangci-lint` complains in the CI

**Review by dduportal — CHANGES_REQUESTED** [review]
image

**Review by dduportal — APPROVED** [review]
LGTM

**Review by ldez — APPROVED** [review]
LGTM

**Pull request merged**